CHANGELOG.md

```diff
@@ -83,3 +83,20 @@ Introducing support for LionWeb 2024.1
 ### Version 0.4.0
 
 * Introduction of the `repo-client` and `repo-client-testing` modules
+
+### Version 0.4.1
+
+* Add support for more APIs in the repo client
```

Comment:
Would it make sense to mention the APIs?

Reply:
Right, I added them